cmd/dlt/accountroles/cmd.go

```diff
@@ -228,15 +231,19 @@ func checkIfRoleExists(clusters []*cmv1.Cluster, role aws.Role) bool {
 	return false
 }
 
-func buildCommand(roleNames []string, policyMap map[string]string) string {
+func buildCommand(roleNames []string, policyMap map[string][]aws.PolicyDetail) string {
 	commands := []string{}
 	for _, roleName := range roleNames {
-		deletePolicy := ""
-		if policyMap[roleName] != "" {
-			policies := strings.Split(policyMap[roleName], ",")
-			for _, policy := range policies {
-				deletePolicy = fmt.Sprintf("\taws iam delete-role-policy --role-name  %s  --policy-name  %s",
-					roleName, policy)
+		policyDetails := policyMap[roleName]
+		for _, policyDetail := range policyDetails {
+			if policyDetail.PolicType == "attached" && policyDetail.PolicyArn != "" {
+				deletePolicy := fmt.Sprintf("\taws iam detach-role-policy --role-name  %s  --policy-arn  %s",
+					roleName, policyDetail.PolicyArn)
+				commands = append(commands, deletePolicy)
+			}
+			if policyDetail.PolicType == "inline" && policyDetail.PolicyName != "" {
```

Comment:
Small NP, would it be worth making `"attached"` and `"inline"` a const in `pkg/aws/client.go` as they are referenced in a few spots.